### DISCUSSION TITLE
사이드 메뉴 티스토리 카테고리 하단 영역이 안떠요

### DISCUSSION THREAD

**Discussion by kkang96** [pronist/hELLO#72] [Q&A] [answered]
**환경 (버그가 발생한 환경을 적어주세요.):**

- 운영체제: [e.g. windows] Window10
- 브라우저: [e.g. chrome, safari] Chrome
- 버전: [e.g. 22] 3.6.1
- 스킨 버전 (스킨 버전은 index.xml 에서 확인): [e.g 3.4.0]

**설명**
버그가 어떻게 발생하는지 알려주세요. 대략적인 버그의 설명을 적어주시면 됩니다.
[e.g. 코드 하이라이팅이 동작하지 않습니다.]

해당 스킨을 사용한 타 블로그를 방문하거나 개발자님 블로그를 보면 티스토리 하단에 운영중인 블로그 영역이 뜨는데,
제 티스토리에서는 뜨지 않아요
따로 설정해주어야하는 부분이 있나요?

**발생 과정**
버그가 어떤 과정을 통해 발생하는지 구체적으로 적어주세요.
[e.g. 테마 변경 버튼을 통해 화이트 모드로 변경시 코드 하이라이팅의 색상이 변하지 않습니다. ...]

스킨 적용 후 처음부터 티스토리 영역 활성화 확인이 불가능합니다.

**스크린샷 및 링크**
버그가 발생한 상황을 가능하다면 스크린샷을 첨부해주세요. 또한 버그가 발생한 모습이 있는 포스트의 주소를 첨부해주시는 것도 아주 좋습니다!

https://ahnsisters.tistory.com/
![image](https://user-images.githubusercontent.com/53862768/158530081-dd4c1e81-0aec-41f9-b7b4-eb9ddc6d43e1.png)

**Comment by pronist** [comment, marked as answer]
@kkang96 **스킨 배포페이지 - 트러블 슈팅 - 더보기 - 티스토리 & 구독 버튼이 안 나와요** 파트 참고부탁드립니다.

**Comment by pronist** [comment]
@kkang96 구독 버튼의 경우 소유자의 블로그에서는 나타나지 않는 것도 참고해주시기 바랍니다.

**Comment by kkang96** [comment]
아 제가 놓친 부분이 있었네요ㅠㅠ 답변 감사합니다 :)
